tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,2647 @@
+          - a6db9d0b3ecb274d90c21e9dde04012b2d13ad8aa0b90e82e7d3b626be67119d
+          - fd31d9c926b5d97ae016535d66baee511dfa3b713b61d8eba2ae5c9929e38ca7
+        payload:
+          type: 5
+          tag: 68656c6c6f20776f726c64
+          data: 5370616d6d696e6720646174612e0a436f756e743a203037323935320a54696d657374616d703a20323032312d30322d31315431303a32333a34392b30313a30300a54697073656c656374696f6e3a203934c2b573
+        nonce: '2102864'
+
+    post-tagged-data-message-request-example-minimal:
+      value:
+        payload:
+          type: 5
+          tag: 68656c6c6f20776f726c64
+          data: 5370616d6d696e6720646174612e0a436f756e743a203037323935320a54696d657374616d703a20323032312d30322d31315431303a32333a34392b30313a30300a54697073656c656374696f6e3a203934c2b573
+
+    post-transaction-message-request-example-minimal:
+      value:
+        payload:
+          type: 0
+          essence:
+            type: 0
+            networkId: 9466822412763346725
+            inputsCommitment: 0e6c2998f5177834ecb3bae1596d5056af76e487386eecb19727465b4be86a79
+            inputs:
+              - type: 0
+                transactionId: 0c78e998f5177834ecb3bae1596d5056af76e487386eecb19727465b4be86a79
+                transactionOutputIndex: 0
+              - type: 0
+                transactionId: 309b169bfa60c8cf80afd730741862d200405d5df7cabfe9b564361dce1f7036
+                transactionOutputIndex: 0
+              - type: 0
+                transactionId: c138ecdc882010245e9574fbc1f1f43cc7c182fa890e46e018365dd49de0f16e
+                transactionOutputIndex: 0
+            outputs:
+              - type: 3
+                amount: 50000000
+                unlockConditions:
+                  - type: 0
+                    address:
+                      type: 0
+                      pubKeyHash: d49931814e6bfbb8e221f1b2936e206e5a6b6abeb9c070f8603814c525c9abe3
+            payload:
+          unlockBlocks:
+            - type: 0
+              signature:
+                type: 0
+                publicKey: 40d0c00fd629a5f528e4bcab4333926a90d06613a3f36a325defcdaf4c9e4092
+                signature: ed4a63a3e1b22c85f9813b00a21c8576ed72c1f5bc939d01fcea7dcd678c2d0d8d80a2499965d8ce5574c9e92aa0d42b72b969bab3a2a35fe7dbc64659a0d703
+            - type: 1
+              reference: 0
+            - type: 1
+              reference: 0
+        nonce: '1020884'
+
+    post-transaction-message-request-example-full:
+      value:
+        protocolVersion: 1
+        parentMessageIds:
+          - 6c5bc96948813faaf6d5a5f0c6bc453bf2e5343664884aea2873041bdc47c4ca
+          - 97e2c4fce0f6e1023c19ed634e37a44744e21a743a79e8f1fc16d31dc8f47481
+          - c107c5f3d6189954c9b1b8e2fa556db4bcf7ad2da717d328ecf6d9a841a5f37d
+          - ea19397d2670f904c4d8c8616ef7a0724321512f1c09f8eb2aeb0cc4299e2441
+        payload:
+          type: 0
+          essence:
+            type: 0
+            networkId: 9466822412763346725
```

Comment:
With https://github.com/iotaledger/iota.go/issues/313, `networkId` from `TransactionEssence` is now of type `string`.
The schemas and examples need to be updated.

Reply:
Done